invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookMetadata/components/RequestNewLoanForm/RequestNewLoanForm.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Form, Transition } from 'semantic-ui-react';
+
+export default class RequestNewLoanForm extends Component {
+  constructor(props) {
+    super(props);
+    this.requestNewLoanForBook = this.props.requestNewLoanForBook.bind(this);
+  }
+
+  requestNewLoan = (itemPid, loan) => {
+    this.requestNewLoanForBook(itemPid, loan);
+  };
+
+  render() {
+    const { docPid, visible } = this.props;
+
+    const loanData = {
+      document_pid: docPid,
+      patron_pid: '111', // FIXME: change this to be able to search for the user
+    };
+
+    //    const volumeOptions = [
+    //      { key: '1', value: '1', text: 'Volume 1' },
+    //      { key: '2', value: '2', text: 'Volume 2' },
+    //    ];
+    //    const editionOptions = [
+    //      { key: '1', value: '1', text: 'Edition 1.2' },
+    //      { key: '2', value: '2', text: 'Edition 1.3' },
+    //    ];
+    const loanPeriodOptions = [
+      { key: '1', value: '1', text: '2 Weeks' },
+      { key: '2', value: '2', text: '3 Weeks' },
+    ];
+    //    const deliveryMethodOptions = [
+    //      { key: '1', value: '1', text: 'Internal Mail' },
+    //      { key: '2', value: '2', text: 'UPS' },
+    //    ];
+
+    return (
+      <Transition visible={visible} animation="scale" duration={500}>
+        <Form>
+          <Form.Group widths="equal">
+            {/*<Form.Select
```

Comment:
Commented out code, its in general a bad practice, because soon we will have no clue if it was working, not working etc. Keep the changes in a personal branch, or create an issue for them.